SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md

```diff
@@ -6,7 +6,7 @@ The CBL-Mariner SPEC files originated from a variety of sources with varying lic
+| Microsoft | [Microsoft MIT License](/LICENSES-AND-NOTICES/LICENSE.md) | application-gateway-kubernetes-ingress <br> azure-iot-sdk-c <br> azure-iotedge <br> azure-storage-cpp <br> bazel <br> blobfuse <br> bmon <br> bpftrace <br> cassandra-cpp-driver <br> ccache <br> cert-manager <br> clamav <br> cloud-hypervisor <br> cloud-init-vmware-guestinfo <br> cmake-fedora <br> coredns-1.6.7 <br> coredns-1.7.0 <br> dbus-x11 <br> debugedit <br> dejavu-fonts <br> distroless-packages <br> doxygen <br> dtc <br> espeak-ng <br> espeakup <br> flannel <br> fluent-bit <br> freefont <br> gflags <br> go-md2man <br> grpc <br> grub2-efi-binary-signed <br> GSL <br> gtk-update-icon-cache <br> helm <br> installkernel <br> ivykis <br> jsonbuilder <br> keda <br> kernel-signed <br> kubernetes-1.18.14 <br> kubernetes-1.18.17 <br> kubernetes-1.19.7 <br> kubernetes-1.19.9 <br> kubernetes-1.20.2 <br> kubernetes-1.20.5 <br> libacvp <br> libconfini <br> libconfuse <br> libiothsm-std <br> libmaxminddb <br> libuv <br> libxml++ <br> lsb-release <br> lttng-consume <br> mariner-release <br> mariner-repos <br> mariner-rpm-macros <br> mm-common <br> moby-buildx <br> moby-cli <br> moby-containerd <br> moby-engine <br> moby-runc <br> msgpack <br> ncompress <br> nlohmann-json <br> nmap <br> nmi <br> node-problem-detector <br> ntopng <br> pcaudiolib <br> pcre2 <br> perl-Test-Warnings <br> perl-Text-Template <br> pigz <br> prebuilt-ca-certificates <br> prebuilt-ca-certificates-base <br> python-cachetools <br> python-cherrypy <br> python-execnet <br> python-logutils <br> python-nocasedict <br> python-pecan <br> python-remoto <br> python-repoze-lru <br> python-routes <br> python-rsa <br> python-sphinxcontrib-websupport <br> python-yamlloader <br> R <br> rocksdb <br> rubygem-asciidoctor <br> rubygem-concurrent-ruby <br> rubygem-console <br> rubygem-deep_merge <br> rubygem-eventmachine <br> rubygem-fiber-local <br> rubygem-hocon <br> rubygem-multi_json <br> rubygem-puppet-resource_api <br> rubygem-rdiscount <br> rubygem-semantic_puppet <br> rubygem-thor <br> shim <br> shim-unsigned <br> shim-unsigned-aarch64 <br> shim-unsigned-x64 <br> span-lite <br> swupdate <br> tinyxml2 <br> toml11 <br> tracelogging <br> vala <br> verity-read-only-root <br> vnstat <br> zstd |
```

Comment:
I'm sorry you had to edit this by hand! If you reach out to me privately with the issue you had, I'll try and find you a fix.

Please also add `cassandra-cpp-driver` to the appropriate location in `SPECS/LICENSES-AND-NOTICES/data/licenses.json`, so we can get this PR passing the license map check.